packages/lab/src/toolbar/ToolbarButton.tsx

```diff
@@ -1,15 +1,16 @@
-import { ElementType, forwardRef } from "react";
-import { Button, ButtonProps, polymorphicRef } from "@jpmorganchase/uitk-core";
+import { forwardRef, Ref } from "react";
+import { Button, ButtonProps } from "@jpmorganchase/uitk-core";
 import "./ToolbarButton.css";
 
-export type ToolbarButtonProps<T extends ElementType = "button"> =
-  ButtonProps<T> & {
-    label?: string;
-  };
+export type ToolbarButtonProps = ButtonProps & {
+  overflowLabel?: string;
+  label?: string;
+};
 
-export const ToolbarButton = forwardRef(function ToolbarButton<
-  T extends ElementType = "button"
->({ label, ...props }: ToolbarButtonProps<T>, forwardedRef: polymorphicRef<T>) {
+export const ToolbarButton = forwardRef(function ToolbarButton(
```

Comment:
I've just noticed, is there a benefit to typing the parameters specifically instead of using the forwardRef's generics?